packages/waku/src/lib/plugins/vite-plugin-deploy-cloudflare.ts

```diff
@@ -62,25 +62,20 @@ export default {
 };
 `;
 
-const getFiles = (dir: string, files: string[] = []): string[] => {
-  const entries = readdirSync(dir, { withFileTypes: true });
-  for (const entry of entries) {
-    const fullPath = path.join(dir, entry.name);
-    if (entry.isDirectory()) {
-      getFiles(fullPath, files);
-    } else {
-      files.push(fullPath);
+const copyFiles = (srcDir: string, destDir: string, extensions: string[]) => {
+  const files = readdirSync(srcDir, { withFileTypes: true });
+  for (const file of files) {
+    const srcPath = path.join(srcDir, file.name);
+    const destPath = path.join(destDir, file.name);
+    if (file.isDirectory()) {
+      mkdirSync(destPath, { recursive: true });
```

Comment:
does this recursive option work on node 18? it might have been with another function call but I think we saw that this got added in node 20 for something else, so it seems good to double check.

Reply:
https://nodejs.org/docs/latest-v18.x/api/fs.html#fsmkdirsyncpath-options

Seems OK with 18.

Reply:
I think it was readdir's recursive option that isn't consistent across node versions. I think mkdir recursive is more stable.